desktop/main-app/src/preload.ts

```diff
@@ -41,6 +41,8 @@ ipcRenderer.once("inject-agora-electron-sdk-addon", () => {
             console.error("[RTC] onerror----", err, msg);
         });
     }
+
+    window.dispatchEvent(new Event("agora-sdk"));
```

Comment:
This will be removed after moving the rest of Agora Electron SDK download logic to flat-rtc-agora-electron